src/cloud/live/sens-data-over.md

```diff
@@ -6,202 +6,132 @@ functional_areas:
+   ```bash
+   git push origin <branch-name>
+   ```
 
-For Pro, when you push to the Git branch, the Integration `master` environment updates. Deploy this branch to Staging and Production. Complete any additional configurations in Staging and Production as needed.
+After the deployment is complete, log in to the Magento Admin panel for the updated environment to verify the settings. Only configured settings display when you use the dump command. Continue to merge any additional configurations to the Staging and Production environments, as needed.
 
 ## Update configurations
 
-If you need to modify or remove any existing configuration settings in the `config.php` file, you will need to modify the file manually with a text editor. After completing edits or removals, you can push it to Git to update.
-
-To add new configurations, modify your environment through the Magento Admin panel and run the command again to generate the file. Any new configurations are appended to the code in the file. Push it to Git to update.
+To add new configurations, modify your environment through the Magento Admin panel and run the command again to generate the file. Any new configurations are appended to the code in the file.
 
 {:.bs-callout-warning}
-While you can manually edit the `config.php` file in the Staging and Production environments, we do not recommend it. The file helps to keep all configurations consistent across all environments. Never delete the `config.php` file to rebuild it. Deleting the file can remove specific configurations and settings required for build and deploy processes.
+While you can manually edit the `config.php` file in the Staging and Production environments, we do **not** recommend it. The file helps to keep all configurations consistent across all environments. Never delete the `config.php` file to rebuild it. Deleting the file can remove specific configurations and settings required for build and deploy processes.
 
-## Migrate configurations
+## Migrate older configurations
 
 If you upgrade to {{site.data.var.ece}} 2.2 or later, you may want to migrate settings from the `config.local.php` file to your new `config.php` file. If the configuration settings in your Magento Admin panel match the contents of the file, follow the instructions to generate and add the `config.php` file.
 
 If they differ, you can append content from the `config.local.php` file to your new `config.php` file:
 
-1. Follow instructions to generate the `config.php` file using the [recommended method](#cloud-config-specific-recomm).
+1. Follow instructions to generate the `config.php` file.
+
 1. Open the `config.php` file and delete the last line.
+
 1. Open the `config.local.php` file and copy the contents.
+
 1. Paste the contents into the `config.php` file, save, and complete adding it to Git.
```

Comment:
```suggestion
1. Paste the contents into the `config.php` file, save, and then commit and push the change to the remote environment.
```

Reply:
Yeah, this is the section that I requested more information for in the MCLOUD ticket. I have no idea what is going on here.